charts/gadget/templates/daemonset.yaml

```diff
@@ -204,6 +208,9 @@ spec:
         - name: debugfs
           hostPath:
             path: /sys/kernel/debug
+        - name: oci-store
```

Comment:
We need `{{- if .Values.config.mountOCIStore }` here as well? CI seems to be [failing](https://github.com/inspektor-gadget/inspektor-gadget/actions/runs/7695905877/job/20969885766#step:7:312) because of this.

Reply:
Resolved.

Reply:
Other than failing checks, is providing conditional statement necessary in the volume block?

Reply:
> is providing conditional statement necessary in the volume block?

I mean it can be confusing to have it in volume block and not mount it in gadget container.